app/views/users/show.html.erb

```diff
@@ -1 +1,31 @@
-<h2> here we will more details about user </h2>
+<main>
+    <div class="user-container">
+        <section class="user">
+            <img src=<%= @user.photo %> alt="User Photo" class="user-photo">
+            <div class="user-details">
+                <h2 class="user-name"><%= @user.name%></h2>
+                <h5 class="user-post-count">Number of posts: <%= @user.post_counter %> </h5>
+            </div>
+        </section>
+        <section class="user-bio">
+            <h3>Bio</h3>
+            <p><%= @user.bio %></p>
+        </section>
+        <section class="user-posts">
+            <h3>Recent Posts</h3>
+            <% @user.three_latest_posts.each do |post| %>
+            <a href="<%=@user.id%>/posts/<%=post.id%>" class="user-post-link">
+                <div class="post">
+                    <h4><%= post.Title %></h4>
+                    <p><%= truncate(post.Text, length: 180) %></p>
+                    <p class="comment-like-count">Comments: <%= post.ComentsCounter %>, Likes: <%= post.likes_counter %></p>
```

Comment:
- [ ] [REQUIRED] Great job so far @AlaaAlsalem , however, while you were trying to resolve the recommendations by the previous reviewer a new bug has been introduced. When I click a user so that it can take me to the show page, I encounter an error
<img width="1166" alt="Screenshot 2024-01-27 at 20 22 32" src="https://github.com/AlaaAlsalem/blogaapp/assets/58692590/901327a2-6d6f-4b6e-b463-0321ade77f0c">

- The error message indicates that there is an attempt to call the likes_counter method on a Post instance, but that method is not defined for the Post model

Reply:
thank you @Whoistolu for your feedback, actually,
![image](https://github.com/AlaaAlsalem/blogaapp/assets/45306358/430b7b85-0d64-4acb-a400-28d2af021c82)
 I could not see  that error from my local environment

Reply:
@AlaaAlsalem 
Did you try for the other users?

Reply:
yes

Reply:
![image](https://github.com/AlaaAlsalem/blogaapp/assets/45306358/3d3ac3fe-51dc-4c42-aa52-e054462ababc)
@Whoistolu 
that is for the other user

Reply:
@AlaaAlsalem 
The error is occuring because the Post model has `LikesCounter` not `likes_counter`
Once you implement that, it will resolve the issue
<img width="1434" alt="Screenshot 2024-01-27 at 21 59 51" src="https://github.com/AlaaAlsalem/blogaapp/assets/58692590/5ec1c2d4-2e9d-4254-b28a-86073fe19268">